contributions/catalog/webapps/templates/contribute/contribute.html

```diff
@@ -24,7 +24,8 @@
 <div class="container-fluid">
     <div class="topnav" id="myTopnav">
         <a href="{{ url_for('contribute.home') }}">Contribution Catalog</a>
-        <a class="active" href="{{ url_for('contribute.create') }}">Submit</a>
+        <a href="{{ url_for('contribute.create') }}">Submit</a>
+        <a>hello, @{{ user }}</a>
```

Comment:
```suggestion
        <a>Hello, {{ user }}</a>
```
Also, please move this (Hello, Name) to right side of the header. Thanks.